includes/api/class-wc-admin-rest-reports-customers-controller.php

```diff
@@ -402,7 +404,7 @@ public function get_collection_params() {
 		);
 		$params['total_spend_between']     = array(
 			'description'       => __( 'Limit response to objects with a total order spend between two given numbers.', 'wc-admin' ),
-			'type'              => 'array',
+			'type'              => 'string',
```

Comment:
Curious if there's a reason we're opting to use strings here and parse inside `normalize_between_params` instead of typing it as an array.  Is this because we don't know if they've filled in both between fields?

```php
		$params['total_spend_between']     = array(
			'description'       => __( 'Limit response to objects with a total order spend between two given numbers.', 'wc-admin' ),
			'type'              => 'array',
			'items'             => array(
				'type' => 'float',
			),
			'validate_callback' => 'rest_validate_request_arg',
		);
```

Reply:
I went with string parsing here because the in its current form, the advanced number filter produces a comma delimited value, not an array. I'll take a look and see how difficult it might be to change that (or parse on the JS side) since I agree that its cleaner to accept an array at the endpoint.

Reply:
Switched back to an array in https://github.com/woocommerce/wc-admin/pull/1260/commits/ad9cbf4f6e3264e2d06fbea2f80778fc35eb4e35.

Reply:
I'll open a separate PR to switch the numeric filter to generate an array.